pkg/common/path_test.go

```diff
@@ -78,6 +82,18 @@ spec:
     image: k8s.gcr.io/pause:2.0
 `)
 
+func buildMultiResourceConfig(configs ...[]byte) []byte {
+	r := []byte{}
+	for i, config := range configs {
+		if i > 0 {
+			r = append(r, []byte(configSeparator)...)
+			// r = append(r, []byte(newline)...)
```

Comment:
Is this something that should be included or should we just delete it?

Reply:
Fixed